### DISCUSSION TITLE
[Bug]: Export rename 404s

### DISCUSSION THREAD

**Discussion by yvdorofeev** [blakeblackshear/frigate#14304] [Report a Bug] [answered]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [ ] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

Attempting to rename an export fails with error:
`Failed to rename export: Request failed with status code 404`

Download works, but rename doesn't

### Steps to reproduce

1. Export video
2. Go to exports
3. Click rename button and observe
...

### Version

0.14.1

### Frigate config file

```yaml
n/a
```

### docker-compose file or Docker CLI command

```yaml
n/a
```

### Relevant Frigate log output

```shell
n/a
```

### Relevant go2rtc log output

```shell
n/a
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

Amcrest

### Screenshots of the Frigate UI's System metrics pages

n/a

**Comment by yvdorofeev** [comment]
Another note: When exporting I used "Select from timeline" option

**Comment by NickM-27** [comment, marked as answer]
thanks, this has been fixed for the next version already
